tests/test_storage.py

```diff
@@ -22,6 +22,8 @@ def test_read_array(tests_root, vocab_array_tuple):
 
 
 def test_mmap_array(tests_root, vocab_array_tuple):
+    if sys.byteorder == "big":
```

Comment:
Use the `skipif` decorator for these cases:

https://docs.pytest.org/en/latest/skipping.html#id1